GCTC-NTGC/gc-digital-talent
# Remove strings from fr.json not in en.json

Resolves #4489.

## Description
Removes unused strings from fr.json files. Related to https://github.com/GCTC-NTGC/gc-digital-talent/pull/4920.

## Summary

- admin: 717-651 = 66
- common: 512-472 = 40
- indigenousapprenticeship: 93-89 = 4
- talentsearch: 714-621 = 93

203 unused strings removed from fr.json files! ✂️ 

## Steps to test

1. Delete all **en.json**, **newTranslations.json**, and **untranslated.json** files from within **/frontend** (these are untracked files and could have leftover work from other branches)
2. Extract admin `npm run intl-extract --workspace=admin`
3. Extract common `npm run intl-extract --workspace=common`
4. Extract indigenousapprenticeship `npm run intl-extract --workspace=indigenousapprenticeship`
5. Extract talentsearch `npm run intl-extract --workspace=talentsearch`
6. Check admin `npm run check-intl-admin --workspace=common`
7. Check common `npm run check-intl-common --workspace=common`
8. Check indigenousapprenticeship `npm run check-intl-indigenousapprenticeship --workspace=common`
9. Check talentsearch `npm run check-intl-talentsearch --workspace=common`
10. Verify there are no **newTranslations.json** or **untranslated.json** files generated in **/frontend**
11. Verify strings removed in fr.json files are not present in codebase by searching all files for their IDs
